edxval/tests/test_api.py

```diff
@@ -334,6 +334,43 @@ def test_get_videos_for_ids_duplicates(self):
         videos = list(api.get_videos_for_ids([edx_video_id, edx_video_id]))
         self.assertEqual(len(videos), 1)
 
+    def test_get_videos_for_ids_sort(self):
+        fish_id = constants.VIDEO_DICT_FISH["edx_video_id"]
+        star_id = constants.VIDEO_DICT_STAR["edx_video_id"]
+        other_id = "other-video"
+        Video.objects.create(**constants.VIDEO_DICT_STAR)
+        Video.objects.create(
+            client_video_id="other video",
+            duration=555.0,
+            edx_video_id=other_id
+        )
+
+        def check_sort(sort_field, asc_expected_ids):
+            """
```

Comment:
What does asc_expected_ids mean? Does it mean we are expecting an ascending list of ids?